src/Apps/ArtQuiz/Components/ArtQuizResultsTabs.tsx

```diff
@@ -22,6 +28,16 @@ export const ArtQuizResultsTabs: FC = () => {
     })
   }
 
+  const calculatePerArtworkRecommendationsLimit = useCallback(() => {
+    if (savedQuizArtworksCount <= 1) return 100
+    if (savedQuizArtworksCount <= 3) return 8
+    return 4
+  }, [savedQuizArtworksCount])
```

Comment:
This doesn't need to be in a callback and can just be inlined in the memo

Reply:
ohhh... good call.